packages/tabs/src/tabs.js

```diff
@@ -161,8 +161,9 @@ export default class Tabs {
 	 * @returns {Array}
 	 */
 	getTabLinksAndList(tabArea) {
-		const tabLinks = tabArea.querySelectorAll('.tab-list [role="tab"]');
-		const tabList = tabArea.querySelector('.tab-list');
+		const tabParent = tabArea.querySelector('.tab-control');
```

Comment:
@xavortm do we want to check for tabParent existence here?

Reply:
Likely want to check for all of these selectors' existence where needed.

Reply:
As per our conversation in Slack, this is a larger refactor for the safety checks. I think you've resolved the issue at hand. Closing for now.